File: python_basic_knowledge/python_speciality/demo_request/demo_1.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
# @Author: yanzhipeng
# @Date:   2019-10-16 21:56:19
# @Last Modified by:   yanzhipeng
# @Last Modified time: 2019-10-17 13:23:23

import requests

r = requests.get("https://api.github.com/user", auth=('user', 'pass'))
print(r)
print(r.status_code)
print(r.headers['content-type'])
print(r.encoding)
print(r.text)
print("***" * 3)
print(r.json)

File: goLang_basic_knowledge/review/201910/1024/demo_1.go
/*
* @Author: yanzhipeng
* @Date:   2019-10-24 18:14:14
* @Last Modified by:   yanzhipeng
* @Last Modified time: 2019-10-24 20:11:34
*/

package main

import (
	"net/http"
)

func main() {

	http.Handle("/", http.FileServer(http.Dir(".")))
	http.ListenAndServe(":8980", nil)
}